src/components/atoms/typography/ms-typography.ts

```diff
@@ -0,0 +1,57 @@
+import { LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+
+import { WithTheme } from '~/mixins'
+
+import { msCheckboxStyles } from './ms-typography.styles'
+
+@customElement('ms-typography')
+export class MSTypography extends WithTheme(LitElement) {
+  static styles = msCheckboxStyles
+
+  @property({ type: String })
+  id = ''
+
+  //@TODO: set to specific strings
+  // 'center' | 'inherit' | 'justify' | 'left' | 'right'
+
+  @property({ type: String })
+  align = 'inherit'
+
+  @property({ type: String })
+  color = ''
+
+  @property({ type: String })
+  weight = ''
+
+  @property({ type: String })
+  variant = ''
+
+  render() {
+    this.style.textAlign = this.align
+    this.style.fontWeight = this.weight
+    this.style.color = this.color
+
+    return this.createTypography()
+  }
+
+  createTypography() {
+    if (!this.variant && this.variant === '') return
```

Comment:
This guard clause could be improved
```suggestion
    if (!this.variant && ['h1','h2'....].include(this.variant)) return
```
to avoid strange tag elements